polygon/rest/aggs.py

```diff
@@ -50,3 +49,82 @@ def get_aggs(
             deserializer=Agg.from_dict,
             raw=raw,
         )
+
+    def get_grouped_daily_aggs(
```

Comment:
Should this be named list_grouped_daily_aggs? I've been naming mine `list` when it returns a results array and `get` when it just returns one object. We haven't discussed naming conventions though so what do you think? (the go client isn't consistent on this either).

Reply:
I think `list` is for iterators that use "next_url" pagination, `get` is for arrays. This one should stay as is, which is `get`.